__tests__/app.test.js

```diff
@@ -75,3 +75,41 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("GET /api/articles", () => {
+  test("200: Returns an array of articles objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toHaveLength(13);
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("400: Responds with 'Bad request' when giving an invalid endpoint", () => {
```

Comment:
a good test but rather than an invalid endpoint this is an invalid article id :)

Reply:
we should also check the right response is sent if someone tries to access a non-existent but valid article like number 1000